apertium-uzb.uzb.lexc

```diff
@@ -28799,6 +28798,7 @@ top%-toza:top%-toza A2 ; ! 0.9931308754372602
 top:top VERB-TV ; ! "to find"
 top:top V-TV ; ! 0.9999174924832647
 top:top N1 ; ! 0.9994055374348901
+topish:topish V-IV ; ! "to find"
```

Comment:
Should be `top:top V-TV`, probably.  Unless you mean "to help find / find together" (the cooperative)?